### ISSUE TITLE
Error index Fulltext (Undefined offset) on Magento 2.1

### ISSUE THREAD

**Issue by DogSports** [Smile-SA/elasticsuite#92]
Hello,

With Magento 2.1 and Elasticsearch 2.2.dev, I have this problem.

```
php bin/magento indexer:reindex catalogsearch_fulltext

Catalog Search indexer process unknown error:
Notice: Undefined offset: 153 in /var/www/vhosts/.../httpdocs/vendor/smile/elasticsuite/src/module-elasticsuite-catalog/Model/Product/Indexer/Fulltext/Datasource/AttributeData.php on line 155
```
I think this is a bug in the html code  for product description. How to find the offending product?

Thank you for your help.

**Comment by romainruaud** [comment]
Hello @DogSports , imho this issue is fixed by the last PR #103 in the 2.2x branch.

Could you confirm this ?

Regards,

**Comment by romainruaud** [comment]
I close this issue. This should have been fixed by #103 as said previously.

Feel free to reopen it if still happening.
